x/pairing/client/cli/tx_stake_provider.go

```diff
@@ -100,7 +100,8 @@ func CmdStakeProvider() *cobra.Command {
 				commission,
 			)
 
-			if msg.DelegateLimit.Denom != commontypes.TokenDenom {
+			if msg.DelegateLimit.Denom != commontypes.TokenDenom ||
```

Comment:
move this to msg_server

Reply:
Fixed in 9140f41f2dc52a2dfd2902c4170f53baed67704b